src/plugins/intel_cpu/src/nodes/random_uniform.cpp

```diff
@@ -39,9 +39,12 @@ RandomUniform::RandomUniform(const std::shared_ptr<ov::Node>& op, const GraphCon
     auto rnd_op = as_type_ptr<op::v8::RandomUniform>(op);
     m_global_seed = rnd_op->get_global_seed();
     m_op_seed = rnd_op->get_op_seed();
-
     m_output_prc = op->get_output_element_type(0);
 
+    if (rnd_op->get_alignment() != op::PhilloxAlignment::TENSORFLOW) {
```

Comment:
Please move it to the fn `isSupportedOperation`.

Reply:
Done!